codes/python/monitor.py

```diff
@@ -18,29 +18,47 @@ def get_disk_space():
     pass
 
 
-def backup(sub_files):
+def backup(sub_files, own=False):
     # files = ["gps_binex.log", "weather_data.log", "thermostat.log", "solar.log", ]
-    source = "/media/mmcblk0p1/logs"
+    time_now = datetime.datetime.now()
+    time_now = str(time_now.year) + "_" + str(time_now.month) + "_" + \
+        str(time_now.day) + "_" + str(time_now.hour) + str(time_now.minute)
+    if own:
+        new_name = sub_files.split("/")
+        new_name.insert(-1, "trashes")
+        # print(new_name[:-1])
+        trash = "/".join(new_name[:-1])
+        if os.path.isdir(trash) == False:
+            print(os.path.isdir("-".join(new_name[:-1])))
+            print(trash)
+            os.mkdir(trash)
+        # print(new_name)
+        new_name = "/".join(new_name)
+        shutil.move(sub_files, new_name)
+        return
+    # source = "/media/mmcblk0p1/logs"
     gps = "/media/mmcblk0p1/backups/gps"
```

Comment:
You could save some typing (and later change things easier) by creating a "constant" for the backup directory location and using that everywhere else: 
```
BACKUP_DIR = '/media/mmcblk0p1/backups'
...
photo_backup = BACKUP_DIR + '/photo'
...
```